business-registry-dashboard/app/components/Form/AddNameRequest/Base.vue

```diff
@@ -48,7 +42,7 @@ const validate = (state: NRSchema): FormError[] => {
   alertText.value = ''
 
   const errors = []
-  const phoneValid = z.string().regex(/^\d{3}-\d{3}-\d{4}$/).safeParse(state.phone).success
+  const phoneValid = state.phone.length <= 12
```

Comment:
How are you stripping out the '-' per the ticket description?

Reply:
Great observation! It's actually on line 84 (the logic was already there).

Reply:
This comment is resolved. (I don't have permission to do that myself.)